### PR TITLE
Added interface to the stanford tokenizer

### PR DESCRIPTION
**Pull request by stevenxxiu**
Added interface to the stanford tokenizer, cf https://github.com/nltk/nltk/issues/558

### CODE REVIEW

File: nltk/tokenize/stanford.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+            input_file.flush()
+
+            cmd.append(input_file.name)
+
+            # Run the tagger and get the output.
+            stdout, stderr = java(cmd, classpath=self._stanford_jar,
+                                  stdout=PIPE, stderr=PIPE)
+            stdout = stdout.decode(encoding)
+            if (not compat.PY3) and encoding == 'ascii':
+                stdout = str(stdout)
```
Comment (kmike) [inline nltk/tokenize/stanford.py]: why not use unicode in 2.x?

Comment (stevenxxiu) [inline nltk/tokenize/stanford.py]: I copied some of the code from `nltk.tag.stanford`, but I'm not sure why either.
I've added a commit which removes this conversion in both this and in `nltk.tag.stanford`.
Thanks.

**Comment by stevenbird** [comment]
Please include a usage example (cf nltk.tokenize.treebank).

**Comment by stevenxxiu** [comment]
I've now added a demo similar to that of `nltk.tokenize.treebank` and another one for the 'americanize' option.

**Pull request merged**